server.js

```diff
@@ -23,6 +41,37 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+//GET request 
+app.get('/thoughts', async (req, res) => {
+  const thoughts = await Thought.find()
+    .sort({ createdAt: 'desc' })
+    .limit(20)
+    .exec()
+
+  res.status(200).json(thoughts);
+})
+
+//POST request
+app.post('/thoughts', async (req, res) => {
+  //The try statement allows you to define a block of code to be tested for errors while it is being executed.
+  try {
+    const postThought = await new Thought(req.body).save()
+    res.status(200).json(postThought)
+  }
+  catch (error) {
+    res.status(400).json({
+      success: false,
+      message: 'Could not post thought',
+      errors: error.errors
+    })
+  }
+})
+
+app.patch('thoughts/:id', async (req, res) => {
+  await Thought.updateOne({ _id: req.params.id }, req.body)
+  res.status(200)
+})
```

Comment:
This endpoint doesn't correspond with your frontend. The logic should increment the hearts in your message/thought 👇
```
hearts: {
    type: Number,
    default: 0,
  },
```

Reply:
How do I solve it? :) I have no idea... need help :)

Reply:
Have you looked at Stack Overflow or the recorded material? 😇 Here's a link to all questions with the tag project-happy-thoughts-api -> https://stackoverflow.com/c/technigo/questions/tagged/189

Reply:
It's solved (I think) :) Please review! :D

Reply:
You've forgotten to update the frontend with the URL to your personal API, so no way to test it! The code looks good though

Reply:
Although I can't like even in Postman. Maybe change the default to 0 instead of `() => 0`

Reply:
Your frontend still looks like it has the technigo-url 🤔